pages/agent/cli_artifact.md.erb

```diff
@@ -33,51 +33,24 @@ Example:
    $ export BUILDKITE_S3_SECRET_ACCESS_KEY=yyy
    $ export BUILDKITE_S3_DEFAULT_REGION=eu-central-1 # default is us-east-1
    $ export BUILDKITE_S3_ACL=private # default is public-read
-   $ buildkite-agent artifact upload "log/**/*.log" s3://name-of-your-s3-bucket/$BUILDKITE_JOB_ID`
+   $ buildkite-agent artifact upload "log/**/*.log" s3://name-of-your-s3-bucket/$BUILDKITE_JOB_ID
 
 Options:
 
-   --job                Which job should the artifacts be uploaded to [$BUILDKITE_JOB_ID]
-   --agent-access-token          The access token used to identify the agent [$BUILDKITE_AGENT_ACCESS_TOKEN]
-   --endpoint 'https://agent.buildkite.com/v3'  The agent API endpoint [$BUILDKITE_AGENT_ENDPOINT]
-   --debug              Enable debug mode [$BUILDKITE_AGENT_DEBUG]
-   --no-color              Don't show colors in logging [$BUILDKITE_AGENT_NO_COLOR]
+   --job value                 Which job should the artifacts be uploaded to [$BUILDKITE_JOB_ID]
+   --agent-access-token value  The access token used to identify the agent [$BUILDKITE_AGENT_ACCESS_TOKEN]
+   --endpoint value            The Agent API endpoint (default: "https://agent.buildkite.com/v3") [$BUILDKITE_AGENT_ENDPOINT]
+   --no-color                  Don't show colors in logging [$BUILDKITE_AGENT_NO_COLOR]
+   --debug                     Enable debug mode [$BUILDKITE_AGENT_DEBUG]
+   --debug-http                Enable HTTP debug mode, which dumps all request and response bodies to the log [$BUILDKITE_AGENT_DEBUG_HTTP]
 ```
 
-## Glob path pattern
-
-Glob path patterns are used throughout Buildkite for artifact file paths. We use a [custom globbing function](https://github.com/buildkite/agent/blob/16a9807a91c4dd1388649557108d4092ae4860a4/glob/glob.go) on the backend that expands the patterns available. The path patterns differ slightly when uploading and downloading artifacts. 
-
-The pattern that you define should be designed to match whole strings rather than substrings. 
-Keep in mind while you're writing your path pattern:
-
-- there are two wildcards available that match non-separator characters:
-  + `*` to match a sequence of characters
-  + `?` to match a single character
-- character ranges surrounded by `[]` support the `^` as a negator
-- special characters can be escaped with `\\`
-- multiple paths are separated with `;`
-- surround the pattern with quotes
-
-The source path you supply to the upload command will be replicated exactly at the destination. If you run:
-
-```bash
-buildkite-agent artifact upload "log/test.log"
-```
-
-Buildkite will store the file at `log/test.log`. If you want it to be stored as `test.log` without the full path, then you'll need to change into the file's directory before running your upload command:
```

Comment:
I thought this was important information in the docs - since it's a quirk of the uploader and downloader that you wouldn't expect - but the path the file is stored at is important for downloading.